libs/components/forms/testing/src/checkbox/checkbox-harness.spec.ts

```diff
@@ -239,15 +246,19 @@ describe('Checkbox harness', () => {
     );
   });
 
-  it('should open help inline popover', async () => {
+  it('should open help inline popover and help widget when clicked', async () => {
```

Comment:
Can we make these two separate tests since they are isolated features?

Reply:
went ahead and did the same for other components with the same unit test